api/lib/src/models/tool.dart

```diff
@@ -89,6 +90,7 @@ sealed class Tool with _$Tool {
     @Default('') String name,
     @Default('') String displayIcon,
     @Default(true) bool zoomDependent,
+    @Default(false) bool straightLineTime,
```

Comment:
Why a bool instead of a null able double? for seconds?

Reply:
It's for the checkbox

Reply:
How about unchecking it if the value is null?

Reply:
Or wait it could be useful. How about renaming this to straightLineEnabled and having a second parameter of type double for the seconds